compiler/testData/diagnostics/tests/controlStructures/ForLoopWithExtensionIteratorOnNullable.fir.kt

```diff
@@ -7,9 +7,9 @@ operator fun <K> Container<K>.iterator(): Iterator<K> = null!!
 fun test() {
     val container: Container<String>? = null
     // Error
-    container.<!INAPPLICABLE_CANDIDATE!>iterator<!>()
+    container.<!UNSAFE_CALL!>iterator<!>()
     // for extension iterator, this code compiles, but should not
-    <!INAPPLICABLE_CANDIDATE!>for (s in container) {}<!>
+    <!UNSAFE_CALL!>for (s in container) {}<!>
```

Comment:
For my own record, this should be rather `ITERATOR_ON_NULLABLE`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/controlStructures/ForLoopWithExtensionIteratorOnNullable.kt#L12